test/Schema.js

```diff
@@ -1213,6 +1213,46 @@ describe('Schema tests', function (){
     });
   });
 
+
+  it('Errors when encountering an unknown attribute if errorUnknown is set to true', async function () {
```

Comment:
I think this test should be broken out into two separate tests. I also think we should also test to ensure the error message is correct if possible.

Reply:
Okay, agreed on both your points.